### DISCUSSION TITLE
Implications in Laravel packages

### DISCUSSION THREAD

**Discussion by caouecs** [Laravel-Lang/common#40] [Q&A]
Hello everybody,
for the moment, we are just actors on translations of tags of Laravel Packages, in this project.

But with the inclusion of Laravel Spark, I think it's time to be actor in Laravel Packages, because we have problems of homogeneity with tags. The country names are best example : 

* "Antigua and Barbuda" => "Antigua And Barbuda"
* "Bolivia" => "Bolivia, Plurinational State of"

It's not good for us to translate a lot of tags, and it's not good for users of these packages.

I will see in Laravel Spark, if it's possible to update tags for country names.

**Comment by andrey-helldar** [comment]
Problem...

**Reply by caouecs** [reply]
yes, Spark is not open source
